static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -524,10 +524,17 @@ class Crossword extends React.Component {
         return currentValue ? currentValue : '';
     }
 
-    onClickHiddenInput () {
+    onClickHiddenInput (event) {
         const focussed = this.state.cellInFocus;
 
         this.onSelect(focussed.x, focussed.y);
+
+        /* We need to handle touch seperately as touching an input on iPhone does not fire the
+        click event - listen for a touchStart and preventDefault to avoid calling onSelect twice on
```

Comment:
Might be worth clarifying in the comment that the click event doesn't fire only when the input is already focused?